CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+worker itself:
+```bash
+open /Applications/Docker.app                                        # If you're on Linux, just `docker`
+rabbitmq-server                                                      # You may need `sudo` for this
+celery worker --app=project --loglevel=INFO -Q builds,email,cleanup
+```
+
+### Source Control
+
+#### Workflow
+We use a slightly modified [Git Flow][git-flow] process; as such, we highly recommend that contributors read this 
+[Git Flow cheatsheet][git-flow-cheatsheet] to understand what each branch type is used for and how the workflow manages
+multiple changes and releases in progress at once. In short: 
+
+- There's a `master` branch and a `develop` branch; `master` reflects production code while `develop` is the staging
+  area for new features and bugfixes
+- Consequently, `feature` and `bugfix` branches derive from and merge back into `develop`
+-  `release` branches are cut from `develop` to bundle features and bugfixes and package them into a release prior to 
+   merging back into both `master` and `develop`. 
+
+Taken together as a whole, this process enables development to continue apace in a CI/CD-enabled environment: releases
+themselves do not "stop the world" in terms of integrating changes in the development process.
+
+Once you've got the concepts down, it's recommended to either install [gitflow-avh][git-flow-avh] for conveniently 
+automated branch management, or use [this cheat sheet][git-flow-vanilla-equivalent] as a reference for equivalent 
+vanilla git commands.
+
+There are a few additional practices we adhere to when contributing to Lookit:
+1. **Rebase locally before publishing a feature or bugfix branch.** `git rebase -i develop` is your friend. A clean
+   local commit history means that we can get away with *never* squashing on remote branches; thus preserving optimal
+   behavior for commands like `git rerere` and `git blame`. In fact, commands like `git flow $MYFEATURE pull` already 
+   execute `git pull --rebase` under the hood. Please do your part to keep our commit history clean and meaningful!
+2. The flip side of #1: **do not *ever* try to rebase or amend commits that are already on a remote branch**. No force
+   pushing, _ever_, unless you know _exactly_ what you are doing (if you know what you're doing, you probably don't need
+   to force push!).
+3. **Prepend Feature and Bugfix branches with the relevant issue ID**. For example, `feature/600-new-cool-thing` or 
+   `bugfix/601-horrible-terrible-bug`. It's a small thing, but it helps us better understand what you're working on.
+4. **Use Pull Requests after publishing feature/bugfix branches.** We use PRs for code review. Since our `origin` uses
+   the default strategy of `--no-ff` merges ([configured][merge-methods-github] in GitHub), you can press the merge
+   button or issue a local `git flow ... finish` without *ever* having to force push. 
```

Comment:
this feels a little unnecessarily intimidating. the instructions are roughly "go read some stuff, and install some stuff, and also here are some modifications to what you read" - which is very efficient, but there isn't actually "here's *what to do*" included. I think that might not actually be so complex, right? What someone should do if they want to add a feature is
- make their own fork. 
- make a branch on that fork, off of develop, and name it based on the issue
- make their changes
- including: add their tests!
- rebase on develop locally (+ warnings about rebasing if already on remote branch - but instead of just "NEVER DO THIS" maybe a friendly statement of what to do instead, e.g. create a new local branch off that one, rebase THAT on develop, push THAT up...? or some other recommendation?) 
- push up that feature branch
- make your PR and code review will get started soon!

The stuff about pressing the merge button and --no-ff merges seems like it'd be aimed at a slightly different audience than this, right?

Reply:
See note about changed headings - as for forking workflow, I'll just make a note to target `develop` with PRs since that's really the only unique thing about what we encourage compared to the stuff that's laid out in the links